tests/Unit/ErrorResponseBuilderTest.php

```diff
@@ -44,10 +44,29 @@ public function respondMethodShouldReturnAJsonResponse()
 
         // Act...
         $response = $responseBuilder->respond();
+        $responseArray = json_decode($response->content(), true);
 
         // Assert...
         $this->assertInstanceOf(JsonResponse::class, $response);
         $this->assertEquals($response->status(), 500);
+        $this->assertArrayHasKey('success', $responseArray);
+        $this->assertEquals(false, $responseArray['success']);
+    }
+
+    /**
+     * Test that the [respond] method does not respond with success flag
+     *
+     * @test
+     */
+    public function respondMethodShouldNotOutputSuccessFlagWhenDisabled()
+    {
+        $this->app['config']->set('responder.include_success_flag', false);
+        $responseBuilder = $this->app->make('responder.success');
```

Comment:
Nice catch! Might be worth moving all the setup variables into a local `setUp` but I'll update just this little piece for now